packages/react-i18n/src/i18n.ts

```diff
@@ -548,3 +560,15 @@ function dateTimeFormatter(
 ) {
   return new Intl.DateTimeFormat(locale, options);
 }
+
+function getSeparators(str: string) {
+  let thousandSeparator;
+  let decimalSeparator;
+  for (const char of str) {
```

Comment:
How will that scale to ltr languages?

Reply:
I'm not sure that there are any languages where numbers are written from right to left (I assume that's what you meant). Here is something I found about Arabic:
> Though Arabic words are written and read in RTL (right-to-left) directionality, numbers are read left-to-right (LTR), just as they are in English

https://www.industryarabic.com/numbers-in-arabic/

Would be good to confirm though

Reply:
Yes that's what I meant.
Ok, after digging a bit I can confirm that number are written left-to-right:
> Historically, the Western numbering system is derived from the Hindu-Arabic numbering system; popularized by Muslim mathematicians like Muḥammad ibn Mūsā al-Khwārizmī. When these books were translated from Arabic, however, Europeans who had a left-to-right language, still maintained the Arabic numbering order. Why Europeans did this nobody knowes.

[source](https://www.quora.com/Why-are-numerals-written-left-to-right-in-Arabic)

Also, interestingly: 
> In Standard Arabic, the smaller number should be read first. For example, 1436 would be six-and-thirty-and-four-hundred-and-a-thousand. The ordering now makes sense.